Mathlib/Init/Function.lean

```diff
@@ -50,6 +52,8 @@ theorem comp_const_right (f : β → φ) (b : β) : f ∘ (const α b) = const 
 /-- A function `f : α → β` is called injective if `f x = f y` implies `x = y`. -/
 def injective (f : α → β) : Prop := ∀ ⦃a₁ a₂⦄, f a₁ = f a₂ → a₁ = a₂
 
+#align function.injective Function.injective
```

Comment:
```suggestion
def Injective (f : α → β) : Prop := ∀ ⦃a₁ a₂⦄, f a₁ = f a₂ → a₁ = a₂
```
I agree with mathport here. This is a prop, it should get capitalized under the naming convention.